packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -1,15 +1,38 @@
 import { ISearchBoxStyleProps, ISearchBoxStyles } from 'office-ui-fabric-react/lib/SearchBox';
 
 export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBoxStyles> => {
-  const { theme } = props;
+  const { theme, underlined } = props;
   const { effects, palette } = theme;
 
   return {
     root: [
       {
         borderRadius: effects.roundedCorner2,
-        borderColor: palette.neutralSecondaryAlt
+        borderColor: palette.neutralTertiary
+      },
+      underlined && {
+        borderRadius: '0px'
       }
-    ]
+    ],
+    iconContainer: {
+      color: palette.themeDarkAlt
+    },
+    clearButton: {
+      selectors: {
+        '&:hover': {
+          selectors: {
+            '.ms-Button': {
+              backgroundColor: 'transparent'
+            },
+            '.ms-Button-icon': {
+              color: palette.black
```

Comment:
@betrue-final-final are we using `black` for hover state? could this be a toolkit bug? I don't remember any hover states of any other component to be black.

Reply:
I have spoken with Ben and changed it to neutralPrimary. Thanks @Vitalius1 !